src/test/fixtures/profiles/gecko-profile.js

```diff
@@ -592,11 +592,13 @@ function _createGeckoThread(): GeckoThread {
         ],
 
         // INSERT NEW MARKERS HERE
+        // Be careful to make sure that the marker below always have a time
```

Comment:
Since this is a test fixture we could write a quick assertion below that the markers are sorted, and not have to rely on a comment.

Reply:
testing the tests ! I like that

Reply:
As discussed on slack, tctually it's possible in gecko that the markers aren't sorted, and we have some out-of-order markers here to be able to test this. So I'm not adding an assertion finally.